test/models/test_gpt.py

```diff
@@ -97,94 +127,152 @@ def decoder(decoder_layer, num_layers):
 
 
 @pytest.fixture
-def gpt(decoder, in_seq_len, out_seq_len, emb_dim):
+def mm_decoder(
+    in_token_emb, out_token_emb, in_pos_emb, out_pos_emb, decoder, right_shift
+):
     return MultimodalTransformerDecoder(
-        in_token_emb=nn.Identity(),
-        out_token_emb=nn.Identity(),
-        in_pos_emb=nn.Embedding(in_seq_len, emb_dim),
-        out_pos_emb=nn.Embedding(out_seq_len, emb_dim),
-        decoder=decoder,
+        in_token_emb, out_token_emb, in_pos_emb, out_pos_emb, decoder, right_shift
     )
 
 
 class TestMultimodalTransformerDecoder:
     def _pos_ids(self, x):
-        bs, seq_len, _ = x.shape
+        b, seq_len, _ = x.shape
         pos_ids = torch.arange(seq_len, dtype=torch.long, device=x.device)
-        return pos_ids[None, :]  # (bs, seq_len)
+        return pos_ids[None, :]  # (b, seq_len)
 
-    def test_bad_input(self, gpt):
+    def test_bad_input(self, mm_decoder):
         with pytest.raises(ValueError):
-            gpt()
+            mm_decoder()
 
-    def test_forward_in_modality(self, gpt, in_modality):
-        actual = gpt(in_modality=in_modality, in_pos_ids=self._pos_ids(in_modality))
+    def test_forward_in_modality(self, mm_decoder, in_modality):
+        actual = mm_decoder(
+            in_modality=in_modality, in_pos_ids=self._pos_ids(in_modality)
+        )
         expected = {
             "last_hidden_states": (
                 torch.Size([1, 3, 4]),
-                -0.4598,
-            ),  # (bs, in_seq_len, emb_dim)
+                -0.5538,
```

Comment:
Why do the expected values change here? As far as I can tell the inputs are the same, so is it just the inclusion of RightShift?